libs/ledger-live-common/src/reconciliation.ts

```diff
@@ -442,6 +425,22 @@ export function patchAccount(
       }
       break;
     }
+    default: {
+      const bridge = getAccountBridge(account);
+      const fromResourcesRaw = bridge.fromResourcesRaw;
+      if (
+        fromResourcesRaw &&
+        !areSameResources(
+          account.accountResources,
+          updatedRaw.accountResourcesRaw
+        )
+      ) {
+        next.accountResources = fromResourcesRaw(
+          updatedRaw.accountResourcesRaw
+        );
+        changed = true;
```

Comment:
i would suggest a refinement to avoid having to introduce a field `accountResources` that is quite impacting.

instead the goal would be to try to "move" the code here into a bridge method.

as we can see in the big switch, what we need fundamentally is a way to "mutate" the updatedRaw to apply the conversion, so maybe there is a way to try to convert this into a method?

i'm not 100% sure but at quick look i can thing of something like
if we may do something like:

```
applyReconciliation: (account: Account, updatedRaw: AccountRaw, next: Account) => boolean,
```

where the boolean is that "changed" value.

basically, we have something in the bridge that can be called to apply the extra logic we see here.

on which we can do:

```
const bridgeChanged = bridge.applyReconciliation(account, updatedRaw, next);
if (bridgeChanged) { changed = true }
```

and an example implementation is basically

```
function applyReconciliation(account, updatedRaw, next) {
const algorandAcc = account as AlgorandAccount;
      const algorandUpdatedRaw = updatedRaw as AlgorandAccountRaw;
      if (
        !areSameResources(
          algorandAcc.algorandResources,
          algorandUpdatedRaw.algorandResources
        )
      ) {
        (next as AlgorandAccount).algorandResources = fromAlgorandResourcesRaw(
          algorandUpdatedRaw.algorandResources
        );
        return true;
      }
}

```


this comment feedback is only addressing the topic of `reconciliation.ts`, but then we need to do that exact same exercice for each part. (i didn't think about the part of "serialisation", but maybe we can mutualize the same principle, optimize and iterate the idea shared here)

Reply:
fixed. Thanks

Reply:
Looks like it has been done, i am resolving